lucupy/minimodel/observation.py

```diff
@@ -74,15 +103,27 @@ class ObservationClass(IntEnum):
 
 @dataclass
 class Observation:
-    """
-    Representation of an observation.
+    """Representation of an observation.
     Non-obvious fields are documented below.
-    * id should represent the observation's ID, e.g. GN-2018B-Q-101-123.
-    * internal_id is the key associated with the observation
-    * order refers to the order of the observation in either its group or the program
-    * targets should contain a complete list of all targets associated with the observation,
-      with the base being in the first position
-    * guiding is a map between guide probe resources and their targets
+    Attributes:
+        id (ObservationID): should represent the observation's ID, e.g. GN-2018B-Q-101-123.
+        internal_id (str): is the key associated with the observation
+        order (int): refers to the order of the observation in either its group or the program
+        title (str):
+        site (site): the Site in which the Observation has to be done.
+        status (ObservationStatus):
+        activate (bool): ???
```

Comment:
This should be `active` and it just means if the `Observation` is currently active.